cephfs/permissions.go

```diff
@@ -29,3 +29,12 @@ func (mount *MountInfo) Chown(path string, user uint32, group uint32) error {
 	ret := C.ceph_chown(mount.mount, cPath, C.int(user), C.int(group))
 	return getError(ret)
 }
+
+// Lchown changes the file/directory last access and modification time.
```

Comment:
I may have missed this previously. The doc comment is well formatted, but the content is wrong. Lchown changes the file ownership not times. Above, regular chown has "Chown changes the ownership of a file/directory." So I'd recommend something like "Lchown changes the ownership of a file/directory/etc without following symbolic links."

Reply:
Sure, will do, when I did this, I took reference from its declaration in C, and there, all the function's comment in go-ceph were exactly the same in its C code. Fixing it.

Reply:
done, please check.